prow/integ-suite-kind.sh

```diff
@@ -107,8 +107,12 @@ grep 'model' /proc/cpuinfo
 # Default IP family of the cluster is IPv4
 export IP_FAMILY="${IP_FAMILY:-ipv4}"
 
+#Kind will not have a LoadBalancer, so we need to disable it
+export TEST_ENV=kind
```

Comment:
seems like a regression, we support metallb for all cases today and you are actually disabling it for ipv6 here